app/assets/javascripts/application.js

```diff
@@ -1131,3 +1131,12 @@ function extractIdentifierFromId(domId) {
     idParts.pop();
     return idParts.join('-');
 }
+
+function appendOptionsToDropdown(options, dropdown) {
+    $(options).each(function(index, element) {
+        dropdown.append($('<option />', {
+            value: element[1],
+            text: element[0]
+        }));
+    });
```

Comment:
Best practice in cases like this is to not write to DOM on each iteration, but instead construct a larger HTML block via iteration, then after all iterations finish, append the full HTML block once.  Fewer DOM writes improves speed.

That's not a blocking concern, because the number of iterations is likely always low, but it's worth noting.

Reply:
That's a great point - I'll make that edit.

Reply:
71d66e3